cpp/src/Ice/SSL/SecureTransportTransceiverI.cpp

```diff
@@ -82,23 +82,6 @@ Ice::SSL::SecureTransport::TransceiverI::initialize(IceInternal::Buffer& readBuf
         _connected = true;
     }
 
-    //
-    // Limit the size of packets passed to SSLWrite/SSLRead to avoid
```

Comment:
My understanding is all the transports we provide on macOS and iOS are non-blocking. As a result, there is no reason to limit the amount of data given to SSLWrite/SSLRead.